templates/taxi/car_detail.html

```diff
@@ -1,22 +1,43 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>
-    {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
-  </h1>
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
-  <hr>
-  <ul>
+  <h2>
+    <strong>
+    Model: {{ car.model }}
+    </strong>
+  </h2>
+  <p><strong>Manufacturer:</strong> ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  {% if user not in car.drivers.all %}
+    <form method="post" action="{% url 'taxi:driver-assign' car.pk %}">
+    <h3 class="car-detail-h3">Drivers
+      {% csrf_token %}
+        <button type="submit" name="assign" class="btn btn-success link-to-page">
+          Assign me to this car
+        </button>
+    </h3>
+    </form>
+  {% else %}
+    <form method="post" action="{% url 'taxi:driver-resign' car.pk %}">
+    <h3 class="car-detail-h3">Drivers
+      {% csrf_token %}
+        <button type="submit" name="unassign" class="btn btn-danger link-to-page">
+          Delete me from this car
+        </button>
+    </h3>
+    </form>
+  {% endif %}
+  <ul class="car-detail-list">
     {% for driver in car.drivers.all %}
-      <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
+      <hr>
+      <li><strong>Username: </strong>{{ driver.username }}</li>
+      <li><strong>Full name: </strong>{{ driver.first_name }} {{ driver.last_name }}</li>
+      <li><strong>License number: </strong>{{ driver.license_number }}</li>
     {% endfor %}
   </ul>
+  <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary">
+    Update
+  </a>
+  <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger">
+    Delete
+  </a>
```

Comment:
According to the code style guide, it is recommended to add a `Cancel` button apart from the `Delete` one. The `Cancel` button will lead to the previous page the user was on.